README.md

```diff
@@ -38,19 +38,14 @@ Code:
 
 ;; read train and test datasets
 (def titanic-train
-  (->
-   (ds/dataset "https://github.com/scicloj/metamorph-examples/raw/main/data/titanic/train.csv"
-               {:key-fn keyword
-                :parser-fn :string})))
-                
+  (-> "https://github.com/scicloj/metamorph-examples/raw/main/data/titanic/train.csv"
+      (ds/dataset {:key-fn keyword :parser-fn :string})))
```

Comment:
Just in case, we can also, have this code as the next example.
```suggestion
  (ds/dataset "https://github.com/scicloj/metamorph-examples/raw/main/data/titanic/train.csv" {:key-fn keyword :parser-fn :string}))
```

Reply:
thanks. wasa left over from previous code.